mkdir: cannot create directory ‘out/2020-10-28_17-20-37’: File exists
usage: parallel_preprocessing.py [-h] -n NOTES --var1name VAR1NAME --var2name
                                 VAR2NAME --var1range VAR1RANGE VAR1RANGE
                                 VAR1RANGE --var2range VAR2RANGE VAR2RANGE
                                 VAR2RANGE
parallel_preprocessing.py: error: argument --var1range: expected 3 arguments
sed: can't read varfile.txt: No such file or directory
sed: -e expression #1, char 3: invalid usage of line address 0
usage: parallel_single_run.py [-h] --varnames VARNAMES VARNAMES
parallel_single_run.py: error: argument --varnames: expected 2 arguments
